Solutions/Menes.Specs/Features/LinkCollectionExtensions.feature

```diff
@@ -0,0 +1,27 @@
+﻿Feature: LinkCollectionExtensions
```

Comment:
When I first read this, my question was "do we need some tests for when link resolution fails".

But I think they are missing from specs for the `OpenApiLinkResolver` rather than here.

Reply:
Yes, I think this is part of a larger problem of very patch test coverage, so I've added an issue to track that https://github.com/menes-dotnet/Menes/issues/19